src/components/Exchange/ExchangeTVChart.js

```diff
@@ -151,29 +79,60 @@ export default function ExchangeTVChart(props) {
     setChartToken(tmp);
   }, [swapOption, fromToken, toToken, chainId]);
 
-  const symbol = chartToken ? (chartToken.isWrapped ? chartToken.baseSymbol : chartToken.symbol) : undefined;
-  const marketName = chartToken ? symbol + "_USD" : undefined;
-  const previousMarketName = usePrevious(marketName);
+  // const symbol = chartToken ? (chartToken.isWrapped ? chartToken.baseSymbol : chartToken.symbol) : undefined;
+  // const marketName = chartToken ? symbol + "_USD" : undefined;
+  // const previousMarketName = usePrevious(marketName);
 
   const currentOrders = useMemo(() => {
     if (swapOption === SWAP || !chartToken) {
       return [];
     }
 
-    return orders.filter((order) => {
-      if (order.type === SWAP) {
-        // we can't show non-stable to non-stable swap orders with existing charts
-        // so to avoid users confusion we'll show only long/short orders
-        return false;
-      }
-
-      const indexToken = getToken(chainId, order.indexToken);
-      return order.indexToken === chartToken.address || (chartToken.isNative && indexToken.isWrapped);
-    });
+    return orders
+      .filter((order) => {
+        if (order.type === SWAP) {
+          // we can't show non-stable to non-stable swap orders with existing charts
+          // so to avoid users confusion we'll show only long/short orders
+          return false;
+        }
+
+        const indexToken = getToken(chainId, order.indexToken);
+        return order.indexToken === chartToken.address || (chartToken.isNative && indexToken.isWrapped);
+      })
+      .map((order) => {
+        const indexToken = getToken(chainId, order.indexToken);
+        let tokenSymbol;
+        if (indexToken && indexToken.symbol) {
+          tokenSymbol = indexToken.isWrapped ? indexToken.baseSymbol : indexToken.symbol;
+        }
+        const title = `${order.type === INCREASE ? t`Inc.` : t`Dec.`} ${tokenSymbol} ${
+          order.isLong ? t`Long` : t`Short`
+        }`;
+        return { title, price: parseFloat(formatAmount(order.triggerPrice, USD_DECIMALS, 2)) };
+      });
   }, [orders, chartToken, swapOption, chainId]);
 
+  const currentPositions = useMemo(() => {
+    if (!positions || !chartToken) {
+      return [];
+    }
+    return positions
+      .filter((p) => p.indexToken.address === chartToken.address)
+      .map((position) => {
+        return {
+          open: {
+            price: parseFloat(formatAmount(position.averagePrice, USD_DECIMALS, 2)),
+            title: `Open ${position.indexToken.symbol} ${position.isLong ? "Long" : "Short"}`,
+          },
+          liq: {
```

Comment:
might be better to use full name like "liquidationPrice" for concistency? i guess we don't abbreviate it in other places

Reply:
Sure!